keep/api/routes/incidents.py

```diff
@@ -812,15 +836,24 @@ def change_incident_status(
             end_time = datetime.now(tz=timezone.utc)
             incident.end_time = end_time
 
-        incident.assignee = authenticated_entity.email
-        incident.status = change.status.value
+        if incident.assignee != authenticated_entity.email:
+            incident.assignee = authenticated_entity.email
+            add_audit(
+                tenant_id,
+                str(incident_id),
+                authenticated_entity.email,
+                ActionType.INCIDENT_ASSIGN,
+                f"Incident self-assigned to {authenticated_entity.email}",
+            )
+
         add_audit(
             tenant_id,
             str(incident_id),
             authenticated_entity.email,
             ActionType.INCIDENT_STATUS_CHANGE,
-            f"Incident status changed from {incident.status} to {change.status.value} by {authenticated_entity.email}",
+            f"Incident status changed from {incident.status} to {change.status.value}",
```

Comment:
why did you decide to remove the 'by {email}'? @talboren

Reply:
because it's the initiator now, see the screenshot